lab_1/main.py

```diff
@@ -69,7 +89,14 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
```

Comment:
There should be checks for input arguments. Smth like:
`if not isinstance(input_arg_name, expected_type):`
`do something here`